src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -722,4 +722,47 @@ For more information, visit http://docs.nuget.org/docs/reference/command-line-re
   <data name="Error_NoMatchingCertificate" xml:space="preserve">
     <value>The package signature did not match any of the allowed certificate fingerprints.</value>
   </data>
+  <data name="ArgumentCannotBeNullOrEmpty" xml:space="preserve">
+    <value>The argument cannot be null or empty.</value>
```

Comment:
Don't we have a null or empty already? 
Also include the argument name,

Reply:
We didn't in this assembly. Also, there's no need to add the argument since it is being passed to the exception and it adds it to the message. This is consistent with any other of our assemblies.